docs/en/setup/backend/dynamic-config-nacos.md

```diff
@@ -1,6 +1,10 @@
 # Dynamic Configuration Nacos Implementation
 
-[Nacos](https://github.com/alibaba/nacos) is also supported as a Dynamic Configuration Center (DCC). To use it, please configure it as follows:
+[Nacos](https://github.com/alibaba/nacos) is also supported as a Dynamic Configuration Center (DCC).
+
+**The Nacos client has upgraded to 2.x, so required Nacos Server version is  2.x**.
```

Comment:
```suggestion
[Nacos](https://github.com/alibaba/nacos) 2.x is also supported as a Dynamic Configuration Center (DCC).
```

SkyWalking docs are versionized. So, this docs only show up when merged commit included version releases.